cluster/jbuild

```diff
@@ -0,0 +1,40 @@
+(* -*- tuareg -*- *)
+#require "unix"
+
+let flags = function
+| [] -> ""
+| pkgs ->
+  let cmd = "ocamlfind ocamlc -verbose" ^ (
+    List.fold_left (fun acc pkg -> acc ^ " -package " ^ pkg) "" pkgs
+  ) in
+  let ic = Unix.open_process_in
+    (cmd ^ " | grep -oEe '-ppx? (\"([^\"\\]|\\.)+\"|\\w+)'")
+  in
+  let rec go ic acc =
+    try go ic (acc ^ " " ^ input_line ic) with End_of_file -> close_in ic; acc
+  in
+  go ic ""
+
+let rewriters_ppx = ["ppx_deriving_rpc"; "ppx_sexp_conv"]
+let rewriters_camlp4 = ["rpclib.idl -syntax camlp4o"]
+
+let coverage_rewriter = ""
+(* (preprocess (pps)) doesn't work with camlp4 and  the other ppx derivers,
+ it complains about missing rpc_of_t *)
+let rewriters_ppx =
+  let is_coverage = try Unix.getenv "BISECT_ENABLE" = "YES" with Not_found -> false in
+  if is_coverage then "bisect_ppx" :: rewriters_ppx else rewriters_ppx
+
+
+let () = Printf.ksprintf Jbuild_plugin.V1.send {|
+(jbuild_version 1)
+
+(library
+ ((name xcp_cluster)
+  (public_name xcp.cluster)
+  (modules (:standard))
+  (flags (:standard -w -39 %s))
+  (libraries (xcp threads rpclib))
+  (wrapped false)
```

Comment:
We should start wrapping `xcp-idl` libraries

Reply:
Please add the executable rules for `cluster_client` and call it `cluster_cli` in line with the others. Do not add a `public_name` or an alias. If https://github.com/xapi-project/xcp-idl/pull/198 is merged before this PR, please update the README

Reply:
Ok, I'll update the PR, you are right we can drop cluster_idl.ml in that case.